modules/meeting/app/workers/meetings/exporter.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+module Meetings
+  class Exporter < ::Exports::Exporter
+    self.model = Meeting
+
+    attr_accessor :template
+
+    def self.key
+      :pdf
+    end
+
+    def initialize(meeting, options = {})
+      opts = options[:options] || options
+      super(meeting, opts)
+
+      self.template =
+        if (opts[:template] == "minutes") && OpenProject::FeatureDecisions.minutes_styling_meeting_pdf_active?
+          Meetings::PDF::Minutes::Exporter.new(meeting, opts)
```

Comment:
I think it would encapsulate the functionality better if this check were part of the Exporter itself. 

That would enable calling
```
Meetings::PDF::Minutes::Exporter.active?(opts[:template])
```

And the `active?` method would do the check:

```
def active?(template)
  template == "minutes" && OpenProject::FeatureDecisions.minutes_styling_meeting_pdf_active?
end
```

That way, the magic string "minutes" would not have to be leaked.